### DISCUSSION TITLE
Handling Lookup Script

### DISCUSSION THREAD

**Discussion by 420skywalker420** [serenity-is/Serenity#5671] [Help]
Hi guys,

Does anyone know how to Handle Lookup Scripts, when I follow instructions from Multi Tenant example I have issues with generic fields . Actually what I want to say, does anyone know how to work with Multi Client Lookup Scripts in .net 5 version? There is a lot of difference between version we work on and version on site.

Regards!

**Comment by minhhungit** [comment]
@420skywalker420  .net 5 version

```csharp
public class MultiTenantRowLookupScript<TRow> : RowLookupScript<TRow>
        where TRow : class, IRow, IMultiTenantRow, new()
    {
    // ....
```
